src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/UserFileWithXamlDefaultsInterceptedProjectPropertiesProvider.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Properties
+{
+    [Export("UserFileWithXamlDefaultsWithInterception", typeof(IProjectPropertiesProvider))]
+    [Export(typeof(IProjectPropertiesProvider))]
+    [Export("UserFileWithXamlDefaultsWithInterception", typeof(IProjectInstancePropertiesProvider))]
+    [Export(typeof(IProjectInstancePropertiesProvider))]
+    [ExportMetadata("Name", "UserFileWithXamlDefaultsWithInterception")]
+    [AppliesTo(ProjectCapability.AlwaysAvailable)]
+    internal sealed class UserFileWithXamlDefaultsInterceptedProjectPropertiesProvider : InterceptedProjectPropertiesProviderBase
```

Comment:
>UserFileWithXamlDefaultsInterceptedProjectPropertiesProvider [](start = 26, length = 60)

Should we have a UserFileInterceptedPropertiesProvider and  ProjectFileWithXamlDefaultsInterceptedProjectPropertiesProvider as well? Or maybe always get the one with XamlDefaults and keep it as just the two types you have added?